.gitignore

```diff
@@ -27,6 +27,7 @@ cffi_bin
 
 # Pyenv Stuff
 .python-version
+venv
```

Comment:
Interesting, do you use venv instead of pipenv? I think most of the Core dev team is on Pipenv haha. (In any case, this is probably fine to check in)

Reply:
Yes, I'm using pipenv but Pycharm, which is my IDE, uses it